sdk/iot_deviceupdate/src/client.rs

```diff
@@ -7,8 +7,9 @@ use const_format::formatcp;
 use serde::de::DeserializeOwned;
 use std::sync::Arc;
 
-pub(crate) const API_VERSION: &str = "2021-06-01-preview";
+pub(crate) const API_VERSION: &str = "2022-10-01";
 pub(crate) const API_VERSION_PARAM: &str = formatcp!("api-version={}", API_VERSION);
+pub(crate) const ENDPOINT: &str = "https://api.adu.microsoft.com/.default";
```

Comment:
This isn't what I meant. The endpoint should not be hardcoded at all. It was a parameter before. Why did you remove it? If it's not correct - which I assume it's not - strip the URL down to its hostname and append "/.default". That's what we do in all our other languages most often.

Reply:
hopefully done the way you expect?